apps/examples/err_report_demo/err_report_main.c

```diff
@@ -295,17 +295,35 @@ void error_report_process(int argc, char *argv[])
 		printf("%s", ERR_REPORT_USAGE);
 		return;
 	}
-	if (strcmp(argv[2], "send") == 0) {
+	if (strcmp(argv[2], "test") == 0) {
 		if (argc < 4) {
-			error_report_records(NULL);
+			error_report_test(NULL);
 		} else {
-			error_report_records(argv[3]);
+			error_report_test(argv[3]);
 		}
-	} else if (strcmp(argv[2], "test") == 0) {
+	} else if (strcmp(argv[2], "report_send") == 0) {
 		if (argc < 4) {
-			error_report_test(NULL);
+			error_report_records(NULL);
```

Comment:
argc < 4
is that correct~?

Reply:
Thank you for your comments. Yes, that is consistent with the fact that we pass argv[3] in the else block. Please let me know if I miss anything.